src/bitcoin-cli.cpp

```diff
@@ -142,8 +142,8 @@ Object CallRPC(const string& strMethod, const Array& params)
         throw runtime_error("no response from server");
 
     // Parse reply
-    Value valReply;
-    if (!read_string(strReply, valReply))
+    Value valReply(UniValue::VSTR);
```

Comment:
UniValue required explicit type definition during constructing.
`Value` should be replaced with `UniValue` soon.